src/domUpdates.js

```diff
@@ -21,19 +23,23 @@ const domUpdates = {
   }),
 
   displayCustomerSearchResults: () => {
-    let user = giveUser();
-    user.searchDate = $('.date-input').val().split('-').join('/')
-    user.addAvailableCabinsToBookingSearch()
-    // display error message
-    // also need to make post request if booking is selected
-    domUpdates.populateCustomerSearchResults(user)
+    let customer = giveCustomer();
```

Comment:
I like how you assigned this variable to the return value of the function. Nice!